net2/SysManager.js

```diff
@@ -988,6 +988,17 @@ class SysManager {
     }
     return false;
   }
+  async getBranchUpdateTime(branch) {
+    try {
+      await exec(`git config remote.origin.fetch "+refs/heads/${branch}:refs/remotes/origin/${branch}"`);
```

Comment:
no need to fetch change to local, it may take long time
just query the last update time remotely.